xLights/effects/StateEffect.cpp

```diff
@@ -234,7 +234,11 @@ void StateEffect::RenderState(RenderBuffer &buffer,
 
     Model* model_info = buffer.GetModel();
     if (model_info == nullptr) {
-        return;
+        buffer.cur_model = buffer.cur_model.substr(0, buffer.cur_model.find("/"));
```

Comment:
(I don't know) ... but do you need to make sure buffer != nullptr?

Reply:
not in this case. you hit this when ytou place a state effect on the sub-model (only); not hitting this if you do it at the model level; by then the buffer is alreay avail.